src/Components/ArticlesPageComponents/ArticlesList.jsx

```diff
@@ -0,0 +1,38 @@
+import { useState } from "react";
+import { useEffect } from "react";
+import { getArticles } from "../../api";
+import axios from "axios";
+import ArticleCard from "./ArticleCard";
+
+const ArticlesList =()=>{
+
+const [articlesList, setArticlesList] = useState([])
+const [isLoading, setIsLoading] = useState(true)
+
+const articlesApi = axios.create({
+    baseURL: "https://v2-nc-news-backend-project-northcoders.onrender.com"
+})
+useEffect(()=>{
+   articlesApi.get("/api/articles").then(({data})=>{
+        console.log(data.articles)
+        setIsLoading(false)
+        setArticlesList(data.articles)
+    })
```

Comment:
Make use of the api call you've declared in api.js instead of writing a new one here

Reply:
done!